### ISSUE TITLE
Maker API app dependency

### ISSUE THREAD

**Issue by briansaycocie** [jvmahon/homebridge-hubitat#1]
Hey thanks for creating this plugin! Just wanted to say, this has been working great for me with no issues that I'm aware of yet.

However, I did run into a challenge when configuring Homebridge. Under step 3 (Setting Up Your Config.json file), it states to get the "Get All Devices with Full Details" URL from the MakerAPI. I had trouble finding this URL and looked everywhere in the settings area. It took me a long time to realize I needed to install the Hubitat Maker API built-in app first as it wasn't pre-installed on my Hubitat hub (just got it about a month ago).

Could we add that to the documentation? Or I'm happy to add it and submit a PR :)

**Comment by jvmahon** [comment]
Yes, good idea. I just added a brief description as item #2 on github.  I'll add some captured images in the coming week.

I  plan to expand these instructions. I just made this all available over the last few days, so its still in the early stages!

More to come over the next few weeks. 

You'll notice a lot of logged messages - much of this will go away  - its there for now as this is still "early Alpha" code but I think its running OK for now.

Additional documentation or clarification via a PR is always welcome!
